.github/workflows/project-tests.yml

```
@@ -46,8 +50,25 @@ jobs:
         dotnet restore DfE.FindInformationAcademiesTrusts
         dotnet build DfE.FindInformationAcademiesTrusts -c Release
 
-    - name: Run mutation tests
-      run: dotnet stryker
-
     - name: Ensure there are no pending migrations
       run: dotnet ef migrations has-pending-model-changes --context FiatDbContext --project DfE.FindInformationAcademiesTrusts.Data.FiatDb --startup-project DfE.FindInformationAcademiesTrusts
+
+    - name: Run mutation tests
+      run: dotnet stryker --reporter "markdown"
```

```suggestion
      run: dotnet stryker --reporter "markdown" --reporter "html"
```
Would be great to upload the html report as an artifact too so we can download it and get much more specific info when needed (there's a lot more info in html)